tests/unit/completion/test_models.py

```diff
@@ -544,7 +544,7 @@ def test_bookmark_completion(qtmodeltester, bookmarks):
 ])
 def test_bookmark_completion_delete(qtmodeltester, bookmarks, row, removed):
     """Test deleting a quickmark from the quickmark completion model."""
-    model = miscmodels.bookmark()
+    model = miscmodels.BookMark().populate(info=None)
```

Comment:
ah, I guess this is why `info` is `Optional`. I don't think that ever happens in real life though, so lets pass in a default `CompletionInfo` in all tests.

Reply:
What does a default ```CompletionInfo``` look like?

Reply:
Just `CompletionInfo()`, I would think.

Reply:
This is how ```CompletionInfo()``` looks like:
```
@dataclasses.dataclass
class CompletionInfo:

    """Context passed into all completion functions."""

    config: config.Config
    keyconf: config.KeyConfig
    win_id: int
    cur_tab: 'browsertab.AbstractTab'
```

I don't see an easy way to make a default. It might be possible but it doesn't look simple atleast at first glance.

Reply:
Looks like we already have one :)

https://github.com/qutebrowser/qutebrowser/blob/f088a991380441254e78dba2bdef4309c8b7815e/tests/unit/completion/test_models.py#L230-L235

Reply:
Agreed, not making `CompletionInfo` optional and instead using this `info` fixture should make everything simpler (and get rid of quite a few mypy issues).

I guess it originally was optional to make those tests a bit simpler (for completions which don't use `info` anyways), but let's make them match reality instead.